src/Tizen.NUI.Wearable/src/public/RecyclerView/RecyclerView.cs

```diff
@@ -24,7 +24,7 @@ namespace Tizen.NUI.Wearable
     /// <summary>
     /// [Draft] This class provides a View that can recycle items to improve performance.
     /// </summary>
-    /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+    [Obsolete("This has been deprecated in API12")]
     [EditorBrowsable(EditorBrowsableState.Never)]
     public class RecyclerView : ScrollableBase
     {
```

Comment:
consider marking this first constructor as well

Reply:
Thank you for review.
it has been fixed.